app.js

```diff
@@ -67,7 +67,7 @@ app.use(helmet.noSniff());
 
 if (process.env.HPKP) {
   app.use(helmet.hpkp({
-    maxAge: 1000 * 60 * 60 * 24 * 90,
+    maxAge: 60 * 60 * 24 * 90, // 90 days in seconds
```

Comment:
This is hpkp - something we should actually remove, since we don't use it and it's somewhat dangerous to use.

the hsts time should still be fixed however. 

Set it to at least 18 weeks (a requirement for hsts preloading - https://hstspreload.org/)